_includes/project-new.md

```diff
@@ -51,17 +51,18 @@ Record the short name in your worksheet. <img src="pics/blue_pencil.png">
 {% endif %}
 1. In the _Description_ field, enter `MISO training workshop [Date]`
 1. Choose the amount of _Progress_ that your project has accomplished:
-  * Unknown : I was told to enter this data and I does what they tells me.
-  * Active : We are receiving/have received samples and are actively working on
+  * UNKNOWN : I was told to enter this data and I does what they tells me.
+  * ACTIVE : We are receiving/have received samples and are actively working on
```

Comment:
Can you mention that only active and pending projects will be processed by analysis?